app/components/Nav.tsx

```diff
@@ -82,12 +82,12 @@ export default function Nav() {
 							<svg
 								role="img"
 								viewBox="0 0 24 24"
```

Comment:
Looks like we needed that viewbox change - not sure what the problem there was...

If you update that we can get this up!

```suggestion
								viewBox="0 0 512 512"
```